libs/azure-storage/langchain_azure_storage/document_loaders.py

```diff
@@ -0,0 +1,112 @@
+"""LangChain integrations for Azure Storage."""
+
+from typing import Callable, Iterable, Iterator, Optional, Union
+
+import azure.core.credentials
+import azure.core.credentials_async
+import azure.identity
+from azure.storage.blob import BlobClient, ContainerClient
+from langchain_core.document_loaders import BaseLoader
+from langchain_core.documents.base import Blob, Document
+
+SDK_CREDENTIAL_TYPE = Optional[
```

Comment:
We should make this type start with an underscore (`_SDK_CREDENTIAL_TYPE`) so that it does not appear as part of the public interface and possibly be imported.